### ISSUE TITLE
[a11y] contrast between caption and white to low

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16397]
In the Cupertino theme the contrast between a white background and the caption text style is only 4.3. It needs to be 4.5 or better.

We currently use black54 for this, but according to spec grey700 (for a contrast ratio of 6) should be used.

**Comment by xster** [comment]
What's the theme being referred to here?

**Comment by matthew-carroll** [comment]
@goderbauer can you explain the 4.3 and 4.5 values represent? Also, how can we reproduce these values on our end so that we can be sure we don't violate this again moving forward?

Also, the current values are based on iOS specs. Can you elaborate on what requirements/responsibilities we have to diverge from such specs for accessibility requirements?

**Comment by goderbauer** [comment]
You can check the contrast ratio between two colors here: https://contrast-ratio.com/

**Comment by jonahwilliams** [comment]
We also have the ability to write automated tests to check contrast ratios, see: https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/test/accessibility_test.dart#L473

**Comment by xster** [comment]
Can you point to what in the CupertinoThemeData or CupertinoTextThemeData are you referring to as the captions text style?

**Comment by TahaTesser** [comment]
Not sure where to look and need more explanation such as https://github.com/flutter/flutter/issues/16397#issuecomment-481463678
